MANIFEST.in

```diff
@@ -6,4 +6,4 @@ include requirements.txt
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
 
-include alarm_client/assets/v8.0.0.json
+include alarm_client/assets/v0.8.0.json
```

Comment:
can you change this to be `include alarm_client/assets/*.json` so that this file doesn't need to be updated for each release?